src/quo2/components/community/community_card_view.cljs

```diff
@@ -18,12 +18,13 @@
       :on-press on-press}
      [rn/view
       {:flex 1}
-      [rn/view (style/community-cover-container 40)
+      [rn/view (style/community-cover-container 60)
        [rn/image
-        {:source cover
+        {:source {:uri (get-in images [:banner :uri])}
```

Comment:
I personally prefer that you leave this as `cover` or some other prop and let the page using the component figure out how to get the data to this. It's tying the data structure to the component. @ilmotta @flexsurfer @OmarBasem  wdyt? 🤔

Reply:
No strong opinion @J-Son89, but I find your suggestion reasonable and an improvement.

Reply:
@J-Son89 sorry for the late reply, did not notice the mention.

Generally I prefer to not have long lines/logic when defining props, so yes `cover` looks better.